frontend/src/components/profile/components/diagnoses/diagnoses.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+import moment from 'moment';
+
+import {
+  ButtonColor,
+  ButtonStyleType,
+  ButtonType,
+  Icon,
+  InputColor,
+  InputType,
+  DiagnosisKey,
+} from 'common/enums';
+import { IDiagnosis, IUser } from 'common/interfaces';
+import { RootState } from 'common/types';
+import { Button, Details, TextInput } from 'components/common';
+import { ProfileActionCreator } from 'store/slices';
+
+import styles from './diagnoses.module.scss';
+
+type Props = {
+  user: IUser;
+};
+
+const Diagnoses: React.FC<Props> = ({ user }) => {
+  const dispatch = useDispatch();
+  const { handleSubmit, reset, control, errors } = useForm<IDiagnosis>({
+    defaultValues: {
+      diagnosis: '',
```

Comment:
Create `DEFAULT-VALUES` constant and move in to the `frontend/src/components/profile/components/diagnoses/common/constants`

Use enum key here